mk/kind.mk

```diff
@@ -42,7 +42,7 @@ kind/start: ${KIND_KUBECONFIG_DIR}
 			--name "$(KIND_CLUSTER_NAME)" \
 			--image=kindest/node:$(CI_KUBERNETES_VERSION) \
 			--kubeconfig $(KIND_KUBECONFIG) \
-			--quiet --wait 120s && \
+			--wait 120s && \
```

Comment:
removed `--quiet` intentionally to show kind starting up progress to the user.(when a new node image is getting downloaded means, there is no indication).

Reply:
I suggest you revert this one, as it is not related to the PR, and also controversial.

Note that this will make CI logs full status updates.

Reply:
sure